docs/source/reference/modding/extended.rst

```diff
@@ -0,0 +1,208 @@
+Note that the sound time is slightly higher than the closing animation start, otherwise sound will be played in the end of opening animation in this example.
+
+It is to important to assign a RootCollisionNode for container mesh, otherwise the collision will depend on animation state, what can lead to performance drops.
+
+Advantages of described approach over old animated containers mods:
+
+1. There is no need to attach a script to every container
+
+2. Part of opening animation can be played in the background, so we do not waste a player's time
+
+An example of mod which uses this feature is the `OpenMW Containers Animated`_.
+
+
+Day/night state support
+-----------------------
+
+In OpenMW it is possible to add one or many NiSwitchNodes with a ``NightDaySwitch`` name.
+Every such switch should have at least two child nodes
+(a first one represents the node in the normal mode, the second one - in the night mode, the optional third node - a day mode for interiors).
+
+How such mesh behaves:
+
+1. During day in exteriors it is in the "common" mode (child 0).
+
+2. During night in exteriors it is in the "night" mode (child 0).
+
+3. During day in interiors it is in the "common" mode or "interor day" mode (child 2) depending on weather.
+
+4. During night in interiors it is in the "common" mode.
+
+An actual switch time depends on weather settings in openmw.cfg:
+
+``fallback=Weather_Sunrise_Time,6``
+
+``fallback=Weather_Sunset_Time,18``
+
+``fallback=Weather_Sunrise_Duration,2``
+
+``fallback=Weather_Sunset_Duration,2``
+
+With this example a night mode starts at 20:00 and ends at 6:00.
+
+In interiors we check if the weather is bright to enter the "interior day" mode using the Glare_View setting. If it is >= 0.5, we consider weather as bright:
+
+``fallback=Weather_Clear_Glare_View,1``
+
+``fallback=Weather_Foggy_Glare_View,0.25``
+
+With this example we use the "interior day" mode for Clear weather, but does not use it for Foggy weather.
+
+Keep in mind that after teleportation to another region we still use an old current weather if player did not move to exterior in the new region yet.
+
+This feature can be used to implement a street illumination, glowing windows, etc.
+
+Advantages of described approach over old wods with glowing windows:
+
+1. There is no need to spawn additional objects for day and night mode
+
+2. There is no need to attach a script to every switchable object
+
+An example of mod which uses this feature is the `Glow in the Dahrk`_.
+
+
+Per-group animation files support
+---------------------------------
+
+In original engine it is possible to add a custom animation file to NPC to override some animations (usually idle ones).
+In OpenMW it is possible to override animations via the same file for all actors which use a given basic animation file.
+
+If you want to override animations for all bipedal actors (they use the xbase_anim.nif skeleton), you can put your animations to the
```

Comment:
```suggestion
If you want to override animations for all bipedal actors (which use the xbase_anim.nif skeleton), you can put your animations to the
```

Reply:
The term is "biped" actually. "the" article should be omitted.